man/org.freedesktop.login1.xml

```diff
@@ -236,112 +236,113 @@ node /org/freedesktop/login1 {
     </programlisting>
 
     <refsect2>
+      <variablelist class="dbus-methods" xpath="./para/firstterm/function">
```

Comment:
Why not use an attribute on function instead? Something like `<function defintion="true">...</>`?

Reply:
because that would mean rewriting make-directive-index entirely. The idea that directives are in variablelist is kinda deeply implemented in there. I would have to more-or-less rewrite the whole thing.

Note that I am looking at a completely different and 100% automated approch and I will probably just close this PR.